nixos/modules/config/system-path.nix

```diff
@@ -147,6 +147,7 @@ in
         "/etc/gtk-3.0"
         "/lib" # FIXME: remove and update debug-info.nix
         "/sbin"
+        "/share/backgrounds"
```

Comment:
This list should be reduced and not be increased.

Reply:
This should be in cosmic module.